build.sh

```diff
@@ -11,15 +11,15 @@ file ZKSyncCrypto/libzks/libzkscrypto.a
 rm -rf ZKSyncCrypto.xcarchive
 
 xcodebuild clean archive \
-            ARCHS=x86_64 ONLY_ACTIVE_ARCH=NO \
+            ONLY_ACTIVE_ARCH=NO \
             -configuration Release \
             -sdk iphonesimulator \
             -project ZKSyncCrypto.xcodeproj \
             -scheme ZKSyncCrypto \
             -archivePath ZKSyncCrypto.xcarchive \
             SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
 
-rm -rf x86_64 
+rm -rf x86_64
 mkdir x86_64
```

Comment:
Can you please also update this folder name as it's no longer accurate?

Reply:
Fixed.